content/en/hub/additional-topics/community/MinIO Clusters in TrueNAS SCALE (DOCS-1499)/index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "MinIO Clusters in TrueNAS SCALE"
+description: "How to deploy MinIO Clusters in TrueNAS SCALE."
+weight: 1
+tags: ["SCALE", "MinIO"]
+---
+
+On TrueNAS SCALE 20.12 and later, users can create a MinIO S3 distributed instance to scale out and handle individual TrueNAS node failures. In this example, we will be using four physical TrueNAS systems and creating a distributed cluster.
+
+## Creating datasets
+
+Before getting started, create a dataset or other shared directory on your storage pool to be used for the persistent minIO data (e.g. "/mnt/tank/minio01"). Create datasets across all nodes in advance.
+
+On your first node, go to **Apps > Launch Docker Image**.
+
+<img src="/images/minIO 1 - Launch Docker Image.png">
+<br><br>
+
+First, enter an application name (we picked “minIO-distributed").
+
+Enter “minIO/minIO” as the image name under **Image Repository** and select _Kill existing pods before creating new ones_ under **Update Strategy**.

```

Comment:
Should be all lower-case, minio/minio

Reply:
Apologies. When I looked up the software name, I noticed the "IO" was capitalized so that's how I wrote it. I fixed it in the text, but the image names still have "MinIO" in their names.

Reply:
Just to clarify, it was only that specific minio/minio name that needed to be lower-case. Its a URL / repo name, and the case would normally be all lower. All the other instances of MinIO where it is being used as a proper name is OK to be CamelCase :)

Reply:
Fixed again! Hopefully, I got it right this time. :)

Reply:
There you go! LGTM now :)